includes/CreateWikiRegexConstraint.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Miraheze\CreateWiki;
+
+use StringUtils;
+
+class CreateWikiRegexConstraint {
+
+	/**
+	 * @param array $regexes
+	 * @return bool
+	 */
+	private static function validateRegexes( $regexes ) {
+		foreach ( $regexes as $regex ) {
+			if ( !StringUtils::isValidPCRERegex( $regex ) ) {
+				return false;
+			}
+		}
+
+		return true;
+	}
+
+	/**
+	 * @param array &$regexes
+	 */
+	private static function filterInvalidRegexes( &$regexes ) {
+		foreach ( $regexes as $regex ) {
+			if ( !StringUtils::isValidPCRERegex( $regex ) ) {
+				wfWarn( 'Contains invalid regex.' );
+
+				unset( $regexes[$regex] );
+			}
+		}
+	}
+
+	/**
```

Comment:
Though I can guess this function will remove comments and empty lines, It will be helpful if there is documentation. But please feel free to add or not.

Reply:
Done. Thanks!